R/add_imgur_image.R

```diff
@@ -78,9 +81,8 @@ add_imgur_image <- function(image,
       alt
     }
 
-  glue::glue(
-    "<a href=\"#\"><img src=\"{response_list$link}\" alt=\"{alt_text}\" \\
-     style=\"max-width:512px;width:100%!important;display:block;padding:0;border:0!important;\" border=\"0\"></a>"
-  ) %>%
-    as.character()
+  paste0(
+    "<a href=\"#\"><img src=\"", response_list$link, "\" alt=\"", alt_text, "\"",
```

Comment:
ALL HTML attributes must be `htmltools::htmlEscape(..., attribute = TRUE)`

Reply:
Now done.